### DISCUSSION TITLE
Can anyone please re-upload sumatraPDF-prerelease-12145-64.exe, please?

### DISCUSSION THREAD

**Discussion by Mark-Joy** [sumatrapdfreader/sumatrapdf#2971] [Q&A]
Can anyone please re-upload sumatraPDF-prerelease-12145-64.exe, please?
This is the latest version that support saving highlights in external .smx
Either portable or installation is OK.
I can't believe no one has asked this before...
Search the whole internet, not found!! :(

**Comment by GitHubRulesOK** [comment]
Whilst there are a few rogue copies around the experimental pre-releases are intentionally deleted a few minor issues later (number of backups varies) once multiple updates or languages or bug fixed. for example this bug in 12145 https://github.com/sumatrapdfreader/sumatrapdf/issues/1460

If you need to recompile a specific version you need to backtrack the VS editor to a compatible older version I don't currently have that 3 year old ?? version. but if you use the last Zeniko 3.1 Dev version (SumatraPDF-dev.exe) that works here written in SumatraPDF-dev and shown in 3.2
![image](https://user-images.githubusercontent.com/12754913/191369936-413f8c76-c662-44ae-ae11-f9acabd81d33.png)
 

SMX viewing is still present in 3.2 official release (for compatibility) but the many pre-releases that edited the smx files are long gone, editing was possible with AHK but the best of those required the experimental releases as base method it did not work with full release (possibly could have). Generally for mods or quick text highlighting  I use MS notepad but must admit not done that in a long  while. Had to search for Zenikos build ![favicon](https://user-images.githubusercontent.com/12754913/191376189-8c705ecb-71ac-443d-8bba-2690acc67c34.png) the link is there but invisible on home page :-)
